task_1/cluster_tf/cluster_autoscaler.tf

```diff
@@ -0,0 +1,223 @@
+resource "aws_iam_role_policy_attachment" "cluster_autoscaler" {
```

Comment:
Is there a better way of providing the cluster-autscaler pod with the necessary permissions inside the AWS account other than attaching this policy to all worker nodes?

Reply:
good catch! autoscaler requires own IAM role attached via service account so that ec2 instances does not have elevated privileges, similarly to testservice defined at https://github.com/eugenetaranov/eks_cluster_tf/blob/task_1/task_1/cluster_tf/testservice.tf_